cdk/lib/prism.ts

```diff
@@ -37,7 +43,13 @@ export class Prism extends GuStack {
 			app,
 		});
 
-		const pattern = new GuPlayApp(this, {
+		const { buildIdentifier } = props;
+
+		const filename = `${app}-${buildIdentifier}.deb`;
```

Comment:
The most minor thing in the world, but I get a niggle that we could use a different separator character here? Hyphens are commonly to separate words in the artefct name, making the version pop out will help at a glance, and if someone wants to parse it out for any reason later. I[t looks like](https://github.com/guardian/prism/pull/948/files#diff-752b7c938d22343dca2a196affbde2ca0132902bb317d08f9ad3329da1be341fR17) sbt-native-packager is using `_` as the version delimiter, could we do the same?

Reply:
Can do. Will raise a follow-up PR. Would you think we can formalise the use of `_`? I've been struggling to think how we'd address this TODO - https://github.com/guardian/cdk/blob/890c89e07d6715277ef9db4f01baeafddc7696e8/src/experimental/patterns/ec2-app.ts#L265.